frontend/app/_layout.tsx

```diff
@@ -43,9 +43,7 @@ export default function RootLayout() {
             <Stack.Screen name="onboarding/signup" />
             <Stack.Screen name="CreateRating" />
             <Stack.Screen name="CreateReview" />
-            <Stack.Screen name="PreviewReview" />
             <Stack.Screen name="MediaReviewsPage" />
-            <Stack.Screen name="MediaPage" />
```

Comment:
this page got moved to the tabs layout (but is not shown in the tab bar) so that way the tabs are visible. i thought this would be a bandaid to the "infinitely hitting the back button" loop since you can navigate away via the bottom tabs